deploy.py

```diff
@@ -98,27 +95,62 @@ def create_control_node(config):
             'CAPABILITY_NAMED_IAM'
         ],
     )
-    print("deployed stack!")
+    print("Uploaded deployment spec. CloudFormation will take it from here.")
+
+
+def upload_ssh_key(key_name):
+    """Wait for the secret to be created in the stack, then update it."""
+    id = "{}.pem".format(key_name)
+    while True:
+        try:
+            secrets.describe_secret(SecretId=id)
+            break
+        except:
+            time.sleep(0.25)
```

Comment:
Even when this will work, there should be some way to detect the secret is ready before using it, right?
Or AWS does not provide anything along those lines?